language/ir-testsuite/tests/signer/move_to_extra_arg.mvir

```diff
@@ -0,0 +1,23 @@
+
+module M {
+    resource R { f: bool }
+    t0(s: &signer, a: address) {
+        (copy(s));
+        (R { f: false });
+        (move(a));
+        (move_to<R>());
+        return;
+    }
+}
+// check: POSITIVE_STACK_SIZE_AT_BLOCK_END
+
+//! new-transaction
+
+module N {
+    resource R<T> { f: T }
+    t0<T>(s: &signer, a: address) {
```

Comment:
`T` needed here (same question in a couple of other places below)? Makes the tests a bit  hard to read because `T` is unused here but used above

Reply:
Ah yeah, bad copy paste. I'll remove them